components/textfield/index.css

```
@@ -435,6 +435,7 @@ governing permissions and limitations under the License.
 	/* Firefox and Safari - Remove the margin for input. */
 	margin: 0;
 
+	/* TODO: Determine if the vendor prefix toggling of appearanmce is still needed  */
```

```suggestion
	/* TODO: Determine if the vendor prefix toggling of appearance is still needed  */
```
I saw you created a separate issue to look into this, but just wanted to weigh in. I think both vendor prefixes can probably just be replaced with `appearance: none`. Any necessary prefixing should be done by the build, based on minimum browser settings. The referenced Firefox issue looks like a very old bug where -moz-appearance: none was not working as it should. I think it's still needed to remove browser added widgets for things such as type number fields or search fields [ [ref] ](https://drafts.csswg.org/css-ui/#appearance-switching)